components/cdc/src/observer.rs

```diff
@@ -91,6 +103,58 @@ impl CdcObserver {
             .get(&region_id)
             .cloned()
     }
+
+    fn untrack_raw_ts(&self, raw_region_ts: Vec<RawRegionTs>) {
+        if raw_region_ts.is_empty() {
+            return;
+        }
+        if let Err(e) = self.sched.schedule(Task::RawUntrackTs { raw_region_ts }) {
+            warn!("cdc schedule task failed"; "error" => ?e);
+        }
+    }
+
+    // parse rawkv cmd from CmdBatch Vec and return the max ts of every region.
+    pub fn get_raw_region_ts(&self, cmd_batches: &Vec<CmdBatch>) -> Vec<RawRegionTs> {
+        let mut region_ts = vec![];
```

Comment:
Suggest to check `api_version = V2` to reduce un-necessary scan.

Reply:
`api_version` is added in CdcObserver and filter the non-apiv2 cmd, thus TiDB should not be affect if user does not enable apiv2.